features/step_definitions/fatalities_steps.rb

```diff
@@ -1,3 +1,10 @@
+Given(/^I can navigate to the list of announcements$/) do
```

Comment:
Did you say yesterday that you were going to remove this?

Reply:
Looks like I did, but the removal was in the wrong commit (f78659554ef34e2dd3fc1ae8f1d7c8bfa50fab95 instead of this one).  I'll update that now.